lib/Applify.pm

```diff
@@ -176,6 +194,13 @@ sub print_version {
   printf "%s version %s\n", File::Basename::basename($0), $version;
 }
 
+sub subcommand {
+  my ($self, $name) = (shift, shift);
+  return $self->{subcommand} unless @_;
+  $self->{subcommands}{$name} = { name => $name, desc => $_[0], code => $_[1] };
```

Comment:
I think "code" should be "options" or something.

Reply:
"adaptation"?